security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -184,23 +184,23 @@ func (s *sdsservice) StreamSecrets(stream sds.SecretDiscoveryService_StreamSecre
 			// request's <token, resourceName, Version>, then this request is a confirmation request.
 			// nodeagent stops sending response to envoy in this case.
 			if discReq.VersionInfo != "" && s.st.SecretExist(con.conID, resourceName, token, discReq.VersionInfo) {
-				log.Debugf("Received SDS ACK from %q, connectionID %q, resourceName %q, versionInfo %q\n", discReq.Node.Id, con.conID, resourceName, discReq.VersionInfo)
+				log.Infof("Received SDS ACK from %q, connectionID %q, resourceName %q, versionInfo %q\n", discReq.Node.Id, con.conID, resourceName, discReq.VersionInfo)
```

Comment:
How many logs will we have ? How about a flag to control the logging level in agent ( we can turn it to debug ).

Reply:
We have a flag to control logging level. Adding this block into ingress-sds will enable debugging level log.
          args:
          - --log_output_level
          - 'debug'

Reply:
Set to debugf